### ISSUE TITLE
Wrong documentation in user.hasPassword

### ISSUE THREAD

**Issue by alilloig** [strongloop/loopback#1939]
In the [api docs](https://apidocs.strongloop.com/loopback/#user-prototype-haspassword), it is said that the user.hasPassword method returns a boolean, but the method [actually](https://github.com/strongloop/loopback/blob/6d040a98aea0945a7b52d577b67d94705b0c5a7d/common/models/user.js#L307) returns a promise/callback that will resolve to/return a boolean.

**Comment by jannyHou** [comment]
Thank you @alilloig ! You are right, we would correct it's description as soon as possible!

**Comment by crandmck** [comment]
Also, can take a second arg that is a function - see https://github.com/strongloop/loopback/blob/master/common/models/user.js#L307

**Comment by jannyHou** [comment]
Yeah, thank you @crandmck !

**Comment by jannyHou** [comment]
Since https://github.com/strongloop/loopback/pull/1953 is merged, I am closing issue here. Thanks!

**Comment by Vaelor** [comment]
Still not correct in the newest version.
https://apidocs.strongloop.com/loopback/v/2.27.0/#user-prototype-haspassword
Or am I missing something there, this not being the newest version?

**Comment by jannyHou** [comment]
@Vaelor I think it's merged into 3.x, but the latest apidocs version is still 2.x. I will see cherry pick it into 2.x branch.

**Comment by crandmck** [comment]
Thanks, @jannyHou!

**Comment by gdeckert** [comment]
Still incorrect in 2.29.0.

**Comment by crandmck** [comment]
@jannyHou This is still not fixed. 
Somehow your fix in #1953 got clobbered, because it's no longer in `master`.  Perhaps you could try again?

**Comment by jannyHou** [comment]
@crandmck sure let me try again.

**Comment by crandmck** [comment]
ping...?

**Comment by crandmck** [comment]
Thas was done for 2.x in 8fe77b2a062bf0de945f9b899f9e71af6d23546f
@bajtos Would it be possible to publish 2.29.1 so it gets published to http://apidocs.strongloop.com/loopback/?

**Comment by bajtos** [comment]
Done: `loopback@2.29.1`

**Comment by crandmck** [comment]
Confirmed: https://apidocs.strongloop.com/loopback/v/2.29.1/#user-prototype-haspassword
